Dixit_Client/Service References/DixitService/Dixit_Client.DixitService.SelectCardResult.datasource

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+    This file is automatically generated by Visual Studio .Net. It is 
+    used to store generic object data source configuration information.  
+    Renaming the file extension or editing the content of this file may   
```

Comment:
I think we should put auto generated files to .gitignore. Anyone who pulls the code will generate this locally, lets not pollute the repo.